reference_simulations/compare_with_last.py

```diff
@@ -29,9 +25,14 @@
     if k not in old_files.keys():
         warnings.warn("File {} not in reference data.".format(k))
         continue
+    uv_old = UVData()
+    uv_new = UVData()
     with warnings.catch_warnings():
         warnings.simplefilter('ignore')     # Ignore known_telescopes warning
-        uv_old = h5py.File(old_files[k], 'r')['Data']
-        uv_new = h5py.File(new_files[k], 'r')['Data']
+#        uv_old = h5py.File(old_files[k], 'r')['Data']
+#        uv_new = h5py.File(new_files[k], 'r')['Data']
```

Comment:
Delete these commented lines (and the one 3 lines below) unless there's a compelling reason to keep them